### DISCUSSION TITLE
Review code

### DISCUSSION THREAD

**Discussion by DanilTezin** [community/community#31193] [Other Feature Feedback, Questions, & Ideas]
Hello, please evaluate my code for a test task for the position of junior front-end developer (react). I ask people who have been in development on react for more than a year to disassemble my code
[repository](https://github.com/DanilTezin/car-dealerships-)

**Comment by melvincwng** [comment]
@DanilTezin Hi Danilla, I did a quick review of your React app. Overall looking good, keep up the good work!

One minor suggestion though, I noticed there aren't any unit test cases in your app. It would be good to always write unit tests for your apps, as part of a good software development habit or practice.

Other than that, LGTM! And all the best in your software developer journey 😊, hope you clinch that job!

Hope this reply answers your concerns/question, take care.

**Reply by DanilTezin** [reply]
Thanks a lot!

**Comment by AliRazaSaqib** [comment]
Hey DanilTezin! I just checked out your code, and it looks good. However, as Melvincwng suggested, I recommend that you add a test case to your code.
